stl/inc/ranges

```diff
@@ -1640,9 +1638,7 @@ namespace ranges {
                 : _Current{_STD move(_It._Current)}, _Parent{_It._Parent} {}
             // clang-format on
 
-            _NODISCARD constexpr iterator_t<_Base> base() const& noexcept(
-                is_nothrow_copy_constructible_v<iterator_t<_Base>>) // strengthened
-                requires copyable<iterator_t<_Base>> {
+            _NODISCARD constexpr const iterator_t<_Base>& base() const& /* strengthened */ {

```

Comment:
This says `/* strengthened */` but dropped `noexcept` entirely. Occurs once below.